.github/workflows/ci.yml

```diff
@@ -351,6 +351,7 @@ jobs:
           matrix.profile == 'release' &&
           github.repository == 'denoland/deno' &&
           (github.ref == 'refs/heads/main' || startsWith(github.ref, 'refs/tags/'))))
+        shell: pwsh
```

Comment:
Please double check if you can see anything else that needs to be updated below.